packages/engine/Source/Core/TrackingReferenceFrame.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Constants for identifying well-known tracking reference frames.
+ *
+ * @enum {number}
+ */
+const TrackingReferenceFrame = {
+  /**
+   * Auto-detect algorithm. The reference frame used to track the Entity will
+   * be automatically selected based on its trajectory: near-surface slow moving
+   * objects will be tracked in the entity's local east-north-up reference
+   * frame, while faster objects like satellites will use VVLH (Vehicle Velocity,
+   * Local Horizontal).
+   *
+   * @type {number}
+   * @constant
+   */
+  AUTODETECT: 0,
```

Comment:
Many of our enums use `NONE` for an undefined or default state. Does that make sense here>

Reply:
Most of the enum with a NONE value I've seen are reflecting something meaningful:
- `ArcType.NONE`
- `ExtrapolationType.NONE`
- `Visibility.NONE`
- `HeightReference.NONE`
- ...
I'm not sure a `TrackingReferenceFrame.NONE` value would be of any interest. This would represent an invalid value that should be tested against, and add more noise to the code IMO.
Unless you would prefer me to replace the `TrackingReferenceFrame.AUTODETECT` by `TrackingReferenceFrame.NONE`?